pkg/env/node_index.go

```diff
@@ -7,8 +7,8 @@ var (
 	// that should be scanned by Scanners NodeIndexer
 	NodeIndexHostPath = RegisterSetting("ROX_NODE_INDEX_HOST_PATH", WithDefault("/host"))
 
-	// NodeIndexMappingURL defines the endpoint for the RepositoryScanner to download mapping information from.
-	NodeIndexMappingURL = RegisterSetting("ROX_NODE_INDEX_MAPPING_URL", WithDefault("https://sensor.stackrox.svc/scanner/definitions?file=repo2cpe"))
+	// NodeIndexMappingURLPath defines the Sensor URL path for the RepositoryScanner to download mapping information from.
+	NodeIndexMappingURLPath = RegisterSetting("ROX_NODE_INDEX_MAPPING_URL_PATH", WithDefault("/scanner/definitions?file=repo2cpe"))
```

Comment:
Do we need to go through any kind of deprecation process for this? I actually just don't know our policy

Reply:
~No, I am not deprecating this. This was added recently in 4.7 and it was never added to the Helm charts, so if someone used that setting, they did that only to fix the issues that this fix addresses.
Why? Because the only part that would be changed to make this work is the Sensor host name.  The path that comes after the host address is hardcoded in Sensor, so I see no scenario in which that would be changed (unless someone wanted to point that to the Internet).~

You know what, I will keep the old setting and remove the default. If someone would like to use that - for example to get the definitions from the Internet - then they can. If `ROX_NODE_INDEX_MAPPING_URL` is unset, I will compute the address the way I do it in this PR.

Reply:
PTAL at https://github.com/stackrox/stackrox/pull/14766/commits/9a63299fc4e5fd1ce0b79394791fd68f5d8ec21b